src/controllers/words.js

```diff
@@ -64,18 +64,36 @@ export const getWords = async (req, res, next) => {
       dialects,
       examples,
     };
-    let query = !strict ? searchIgboTextSearch(searchWord, isUsingMainKey) : strictSearchIgboQuery(searchWord);
-    const words = await searchWordUsingIgbo({ query, ...searchQueries });
-    if (!words.length) {
-      query = searchEnglishRegexQuery(regexKeyword);
-      const englishWords = await searchWordUsingEnglish({ query, ...searchQueries });
-      return packageResponse({
-        res,
-        docs: englishWords,
-        model: Word,
-        query,
-        ...rest,
-      });
+    let words;
+    let query;
+    if (searchWord.match(/".*"/) || searchWord.match(/'.*'/)) {
+      const searchWordWithoutQuotes = searchWord
```

Comment:
This could be rewritten as:
```
const searchWordWithoutQuotes = searchWord.replaceAll(/['"]/, '')
```
to remove the quotes instead of iterating over each character